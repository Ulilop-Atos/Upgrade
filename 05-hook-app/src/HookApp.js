import React from 'react'
import { FormWithCustomHook } from './components/02-useEffect/FormWithCustomHook'

export const HookApp = () => {
  return (
    <FormWithCustomHook />
  )
}
