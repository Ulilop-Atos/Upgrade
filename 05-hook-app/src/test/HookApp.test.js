import { shallow } from 'enzyme';
import { HookApp } from "../HookApp";

describe('Tests for <HookApp />', () => { 
  
  test('should display properly', () => { 
    const wrapper =  shallow(<HookApp />);
    expect(wrapper).toMatchSnapshot();
   });

 });