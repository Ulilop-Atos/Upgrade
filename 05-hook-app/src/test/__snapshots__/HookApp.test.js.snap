// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for <HookApp /> should display properly 1`] = `
<Fragment>
  <MainApp />
  <MultipleCustomHooks />
</Fragment>
`;
